apps/glusterfs/health.go

```diff
@@ -0,0 +1,123 @@
+package glusterfs
+
+import (
+	"bytes"
+	"encoding/gob"
+	"encoding/json"
+	"net/http"
+	"time"
+
+	"github.com/heketi/heketi/pkg/glusterfs/api"
+
+	"github.com/boltdb/bolt"
+	"github.com/lpabon/godbc"
+)
+
+var clusterHealth = "ClusterHealth"
+
+type dbchecker struct {
+	CurrentTime time.Time
+}
+
+func NewHealthz() *api.Healthz {
+	health := &api.Healthz{}
+	health.NodesStatus = make([]api.NodeHealthStatus, 0)
+	return health
+}
+
+func (d *dbchecker) Marshal() ([]byte, error) {
+	var buffer bytes.Buffer
+	enc := gob.NewEncoder(&buffer)
+	err := enc.Encode(*d)
+
+	return buffer.Bytes(), err
+}
+
+func (d *dbchecker) Unmarshal(buffer []byte) error {
+	dec := gob.NewDecoder(bytes.NewReader(buffer))
+	return dec.Decode(d)
+}
+
+func (*dbchecker) BucketName() string {
+	return BOLTDB_BUCKET_DBATTRIBUTE
```

Comment:
If this is using the BOLTDB_BUCKET_DBATTRIBUTE bucket why isn't it using the DbAttributeEntry type? I really don't like the idea of multiple types sharing the same bucket. It will just lead to confusion IMO.

Reply:
create new db attribute for healthz